corehq/apps/hqwebapp/templates/hqwebapp/iframe_sso_login_pending.html

```diff
@@ -0,0 +1,26 @@
+{% extends "hqwebapp/base.html" %}
```

Comment:
It seems like SSO logins will never be able to use an iframe, so can we remove the `iframe` from the file names?